pkgs/os-specific/linux/zenmonitor/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, pkgconfig, gtk3, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  name = "zenmonitor";
+  version = "1.2";
```

Comment:
it looks like 1.3 was released a few days ago, would you mind updating it? https://github.com/ocerman/zenmonitor/releases/tag/v1.3